Configuration/TCA/Overrides/sys_file_collection.php

```diff
@@ -49,5 +49,5 @@
 
 \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addToAllTCAtypes(
     'sys_file_collection',
-    '--div--;LLL:EXT:bm_image_gallery/Resources/Private/Language/locallang_be.xlf:bm_image_gallery_tab,' . implode(',', array_keys($temporaryColumns)),
+    '--div--;LLL:EXT:bm_image_gallery/Resources/Private/Language/locallang_be.xlf:bm_image_gallery_tab,' . implode(',', array_keys($temporaryColumns))
```

Comment:
This will not fix the actual bug since the comma must be added to the string. So please change this line to:

`    '--div--;LLL:EXT:bm_image_gallery/Resources/Private/Language/locallang_be.xlf:bm_image_gallery_tab,' . implode(',', array_keys($temporaryColumns)) . ','
`

Reply:
Hi Florian,

thx for your reply. That makes sense and the change has been applied.